xclim/core/utils.py

```diff
@@ -162,3 +164,208 @@ def sfcwind_2_uas_vas(wind: xr.DataArray = None, windfromdir: xr.DataArray = Non
     wind.attrs["units"] = "m s-1"
 
     return uas, vas
+
+
+@declare_units(
+    "%",
+    tas="[temperature]",
+    dtas="[temperature]",
+    huss="[]",
+    ps="[pressure]",
+    ice_thresh="[temperature]",
+)
+def relative_humidity(
+    tas: xr.DataArray,
+    dtas: xr.DataArray = None,
+    huss: xr.DataArray = None,
+    ps: xr.DataArray = None,
+    ice_thresh: str = None,
+    method: str = "sonntag90",
+    invalid_values: str = "clip",
+):
+    """Compute relative humidity from temperature and dewpoint temperature.
+
+    Parameters
+    ----------
+    tas : xr.DataArray
+        Temperature array
+    dtas : xr.DataArray
+        Dewpoint temperature, if specified, "method" is set to "dewpoint".
+    huss : xr.DataArray
+        Specific Humidity
+    ps : xr.DataArray
+        Air Pressure
+    ice_thresh : str
+        Threshold temperature under which to switch to equations in reference to ice instead of water.
+        If None (default) everything is computed with reference to water.
+    method : {"dewpoint", "goffgratch46", "sonntag90", "tetens30", "wmo08"}
+        Which method to use, see notes.
+    invalid_values : {"clip", "fill", None}
+        What to do with values outside the 0-100 range.
+        If "clip" (default), clips everything to 0 - 100,
+        if "fill", replaces values outside the range by np.nan
```

Comment:
`fill`/`fillna` is usually used to fill nan with other values. Maybe choose another name ? `mask` ? `nan` ?

Reply:
Oh yeah "mask"! I didn't like "fill" but couldn't find better ideas, let's go with mask.